packages/types-cloudflare-worker/src/global.ts

```diff
@@ -325,6 +325,21 @@ export interface CloudflareRequestFeatures {
    */
   polish?: string;
 
+  /**
+   * Image resize options:
+   * https://developers.cloudflare.com/images/worker/
+   */
+  image?: {
+    width?: number;
+    height?: number;
+    dpr?: number;
+    fit?: string;
+    quality?: number;
+    format?: string;
+    metadata?: string;
+    rotate?: number;
```

Comment:
Not all of these fields have obvious meaning, please add documentation for all fields so it appears in Intellisense or other code helpers.

Reply:
Excerpts from the Cloudflare documentation added.